README.md

```diff
@@ -74,6 +74,108 @@ The following types are exported:
 - `HandlerEvent`
 - `HandlerResponse`
 
+
+## Netlify Integration Auth Management
+
+To make building on and interacting with third-party APIs as simple and powerful as possible, Netlify provides API secret provisioning and management, powered by OneGraph. It’s enabled on a per-site basis under the [Netlify labs](https://app.netlify.com/user/labs) tab, where you can visually select which services you want to make available for your functions or site builds, and which scopes you need access to.
```

Comment:
Sounds like this location may shift (enable in Netlify Labs, configure elsewhere in the UI)

Reply:
I believe the plan is to get it live while the UI lives inside of the Labs UI, so the readme should be updated to reflect this and then updated again later once we undertake the review